docs/theme/mlflow/static/css/mobile.css

```
@@ -0,0 +1,50 @@
+/*
+ * Custom CSSs for optimizing mobile view
+ *
+ * *** PLEASE DO NOT OVERUSE *** as having two different styles reduces maintainability.
+ */
+
+
+/* Mobile styling */
+@media only screen and (max-width: 768px) {
+    /*
+    * Fix table overflow issue on mobile. Sphinx apply no word wrapping by default,
+    * resulted in showing very wide 1-line cells on mobile.
+    * To apply those CSSs to your table, add `:class wrap-table` to your table
+    */
+    .wy-table-responsive .wrap-table td,
+    .wy-table-responsive .warp-table th {
```

```suggestion
    .wy-table-responsive .wrap-table th {
```

This isn't a typo in the sphinx-tabs plugin source code, is it?